### DISCUSSION TITLE
Can Twinny be used with public, cloud-hosted (non-local) inference services ?

### DISCUSSION THREAD

**Discussion by bdutta** [rjmacarthy/twinny#408] [Q&A]
Have setup Twinny on my LinuxPC and used with Qwen-2.5-7B-Coder model (running on CPU at about 8-10 tk/s) via Ollama, as I don't have a DGPU. Wondering if Twinny could be configured to use Mistral's hosted public Codestral-22B inference service for FIM and Anthropic/Gemini/ChatGPT/Groq/Cerebras free (hosted inference) for chat etc. ? If, how should the provider configuration look like ?

**Comment by rjmacarthy** [comment]
Hey, yes probably it's possible but there is not really technical support on an individual basis from me or any contributors, if you find a way to achieve it please share for others.

Many thanks!
